app/services/idv/session.rb

```diff
@@ -31,8 +31,11 @@ def proofing_started?
       resolution.present? && applicant.present? && resolution.success?
     end
 
-    def profile_from_applicant(applicant)
-      self.profile_id = Profile.create_from_proofer_applicant(applicant, current_user).id
+    def profile_from_applicant(applicant, password)
+      profile = Idv::ProfileFromApplicant.create(applicant, current_user, password)
+      self.profile_id = profile.id
+      cache_encrypted_pii(password)
+      profile
     end
```

Comment:
From what I can tell, the return value (the profile) is not used when this method is called. Perhaps the profile_id assignment should be in its own method: `set_applicant_profile_id`, then instead of calling `idv_session.profile_from_applicant(idv_session.applicant, password)` in `init_questions_and_profile`, you would call both of these methods:

```ruby
idv_session.set_applicant_profile_id(idv_session.applicant, password)
idv_session.cache_encrypted_pii(password)
```
What do you think?